packages/extension-ui/src/components/EditMenuCard.tsx

```diff
@@ -257,9 +252,9 @@ export default styled(EditMenuCard)(
   }
 
   .link-icon {
-    width: 20px;
-    height: 20px;
-    background: ${theme.primaryColor};
+    width: 12px;
```

Comment:
I think 12px is too small, perhaps 14px?

Reply:
Actually, Bryku wanted 16, so changing to 16.